.github/workflows/update_klotho_formula.yaml

```diff
@@ -0,0 +1,75 @@
+name: Update klotho's homebrew formula
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      klotho-tap:
+        description: 'klotho homebrew tap repository'
+        required: false
+        default: 'KlothoPlatform/homebrew-klotho'
+      klotho-release:
+        description: 'the klotho release that the formula will point to'
+        required: false
+jobs:
+  update-brew-formula:
+    runs-on: ubuntu-latest
+    steps:
+      - name: get release name
+        id: releaseName
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          if [ -z "${{ github.event.inputs.klotho-release }}" ]; then
+            tag_name="$(echo "$GITHUB_REF" | cut -d/ -f3)"  # e.g. "v0.5.16"
+            release_name=$(gh release -R ${GITHUB_REPOSITORY} view ${tag_name} --json "name,body" | jq -r '.name')
+            echo "using release ${release_name}"
+          else
+            release_name="${{ github.event.inputs.klotho-release }}"
+          fi
+
+          
+          printf 'release_name=%s\n' "$release_name" >> $GITHUB_OUTPUT
+
+      - name: clone klotho tap
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: gh repo clone ${{ github.event.inputs.klotho-tap || 'KlothoPlatform/homebrew-klotho' }} tap -- -b main --depth 1
+
+      - name: download release artifacts
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          cd tap
+          gh release download ${{ steps.releaseName.outputs.release_name }} -R "${GITHUB_REPOSITORY}" -p 'klotho_*'
+
+      - name: update klotho brew formula
+        run: |
+          cd tap
+          export BINARY_BASE_URL="https://github.com/klothoplatform/klotho/releases/download"
```

Comment:
I assume this is for the `update_formula.sh`? As a minor point, I'd consider moving this line down, to right before the invocation.

I actually had a comment on the previous iteration of "doesn't seem to be used, should this be rm'ed?" partially because of the `-test` in the URL. Given that you specifically updated this line to change the URL, I'm guessing it's intentional — but without that context, I would have probably thought it's not doing anything unless I saw it right next to the line that's consuming it.

Reply:
yes, it's used by the update script. I can move it closer. I figured being part of a distinct step would be enough. 🤷‍♂️